.github/workflows/python-package.yml

```diff
@@ -67,47 +59,41 @@ jobs:
     runs-on: ubuntu-24.04
     steps:
       - uses: actions/checkout@v4
-      - uses: actions/setup-python@v5
-        with:
-          python-version: 3.8
-      - name: Setup
+      - name: Setup uv
+        run: curl -LsSf https://astral.sh/uv/install.sh | sh
```

Comment:
Stick this in a script in `scripts/`, called from `ci-setup.sh`?

Reply:
Done